### ISSUE TITLE
Unclear how to setup a Xamarin UITest project

### ISSUE THREAD

**Issue by edrickleong** [MicrosoftDocs/appcenter-docs#270]
It's unclear which NUnit packages and runners are required to run the Xamarin UITest locally

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8a0474be-6e14-b8e6-b6cf-934a04f0e509
* Version Independent ID: 4a79fc1d-8dca-1c17-718f-c30fda180f05
* Content: [Xamarin.UITest - Visual Studio App Center](https://docs.microsoft.com/en-au/appcenter/test-cloud/uitest/)
* Content Source: [docs/test-cloud/uitest/index.md](https://github.com/MicrosoftDocs/appcenter-docs/blob/live/docs/test-cloud/uitest/index.md)
* Service: **vs-appcenter**
* GitHub Login: @glennwester
* Microsoft Alias: **glwest**

**Comment by jwargo** [comment]
@edrickleong are you asking for help or proposing a doc change?

**Comment by edrickleong** [comment]
@jwargo I'm looking for help, I have had trouble setting up the UI Test to run locally, thanks in advance.

**Comment by jwargo** [comment]
@edrickleong App Center support is provided directly from within the App Center portal. Any time you need help, just click the blue chat button in the lower-right corner of any page and our dedicated support team will respond. For additional information, see the [App Center Help Center](https://intercom.help/appcenter/getting-started/welcome-to-app-center-support).

**Comment by ryanlpoconnell** [comment]
This was not a useful end to a post for anyone.  A doc change would be more useful!

**Comment by jwargo** [comment]
@ryanlpoconnell he said he was looking for help, so that's why I redirected to support. What usually happens is that the support team identifies the issue and helps the customer solve it, then either proposes or makes a doc change to reflect the fix. If you have a change to suggest, I'm happy to make it, or the docs are open source, so feel free to make a PR.

**Comment by ryanlpoconnell** [comment]
@jwargo apologies for the outburst. You are of course correct in your order support redirect.  It's very frustrating to find the very question you have left unanswered.  I will follow this up with a suggested documentation update, but please do not consider it to be directed specifically at yourself.

**Comment by jwargo** [comment]
@ryanlpoconnell no worries, I didn't see it as being directed toward me. Actually, the Test team's processing a doc update right now, so I imagine it will be published soon.  
UPDATE: Expect the doc update to be completed Monday. Sorry for the delay.

**Comment by ryanlpoconnell** [comment]
For those who might read this issue hoping for _an answer_ (albeit perhaps answering a slightly different question).

I found missing information being that the NUnit 2 Test Adapter must be installed in Visual Studio via Tools>Extensions and Updates>Online: search "NUnit 2 Test Adapter".  

I credit this blog: https://www.jimbobbennett.io/ui-testing-your-xamarin-apps/

FYI: until I installed it via Extensions and Updates I was receiving the following Output from the Android UITest project Visual Studio 2017 (Windows): 

```
[26/10/2018 17:36:37 Informational] NUnit VS Adapter 2.1.1.0 discovering test is finished
[26/10/2018 17:36:38 Informational] ========== Discover test finished: 2 found (0:00:01.1763683) ==========
[26/10/2018 17:36:49 Informational] ------ Run test started ------
[26/10/2018 17:36:49 Warning] Could not find test executor with URI 'executor://nunittestexecutor/'.  Make sure that the test executor is installed and supports .net runtime version  .
[26/10/2018 17:36:49 Informational] ========== Run test finished: 0 run (0:00:00.4493378) ==========
```

This step is incredibly difficult to determine as I have not found any Microsoft documentation mentioning this step and the confusion is exacerbated by the fact that a Nuget Package of the exact same name exists.

I concede in hindsight, it makes sense that _something_ has to run the the NUnit tests, but this only smacks you in the face once you've found the answer 8 hours later anyway.
